src/components/vm/disks/diskEdit.jsx

```diff
@@ -60,7 +60,7 @@ const CacheRow = ({ onValueChanged, dialogValues, idPrefix, shutoff }) => {
                    label={_("Cache")}
                    labelIcon={!shutoff &&
                        <Popover bodyContent={_("Machine must be shut off before changing cache mode")}>
-                           <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                           <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230630-122011-699fe1a0-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)